package.json

```diff
@@ -126,6 +127,8 @@
   "license": "BSD-3-Clause",
   "author": "Facebook",
   "jest": {
+    "preset": "react-native",
```

Comment:
Without this line, I get an error when importing `react-native`. This also seems to be the reason why tests fail, as it doesn't allow for inlining of `View` and `Text`.